### ISSUE TITLE
Move/Drag'n'Drop bookmarks not possible when accessibility of other apps is turned on

### ISSUE THREAD

**Issue by Asiaboy89** [kiwibrowser/src.next#890]
System: Android 13
Smartphone: Realme GT2 Pro
Kiwi version used: 112.0.5615.137

For example, I use Nova Launcher, ACR Phone and aodNotify. All three apps need accessibility enabled in the android settings to work. So I turned it on.
![1](https://github.com/kiwibrowser/android/assets/67432501/4c04a511-625a-4d9d-be46-c5a7367948bf)

But that results in the situation, that if **only one** of these apps have enabled the accessibility option (I tested it), I cannot more move any of my bookmarks via the two lines symbol.
![3](https://github.com/kiwibrowser/android/assets/67432501/acb0caec-1292-4e45-a96d-fe5a5b8c6e45)

Instead of the two lines, the Icon changes to three dots, but I cannot move or drag'n'drop the selected bookmark to my desired position.
![2](https://github.com/kiwibrowser/android/assets/67432501/e5914af0-a246-4bf4-9b3f-d514fbc12e14)

As mentioned above, this happens even if only one my apps with accessibility is turned on. Seems like Kiwi has a general problem with Android accessibility settings.

**Comment by Asiaboy89** [comment]
Fixed with Version 116 ;)

**Comment by Asiaboy89** [comment]
Have to reopen it again, because App TapScroll (https://play.google.com/store/apps/details?id=com.scroll.scrolltop.backtotop) causes the same issue with enabled accessibility. When I turn it off for this specific App, then I can move my bookmarks via Drag'n'Drop. At least since Version 116 it works for the above mentioned Apps (Nova, aodNotify, ACR Phone).
